src/components/Modal/TopupTModal/LegacyTopUpModal.tsx

```diff
@@ -0,0 +1,142 @@
+import { FC } from "react"
+import {
+  Button,
+  ListItem,
+  ModalBody,
+  ModalCloseButton,
+  ModalFooter,
+  ModalHeader,
+  UnorderedList,
+  TabPanels,
+  TabPanel,
+  TabList,
+  Tab,
+  Tabs,
+  useColorModeValue,
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  Link,
+  Divider,
+} from "@chakra-ui/react"
+import { Body3, H5 } from "../../Typography"
+import InfoBox from "../../InfoBox"
+import { BaseModalProps } from "../../../types"
+import {
+  ExternalHref,
+  ModalType,
+  StakeType,
+  Token,
+  TopUpType,
+} from "../../../enums"
+import withBaseModal from "../withBaseModal"
+import { TokenAmountForm } from "../../Forms"
+import { useTokenBalance } from "../../../hooks/useTokenBalance"
+import { StakingContractLearnMore } from "../../ExternalLink"
+import { useModal } from "../../../hooks/useModal"
+import { StakeData } from "../../../types/staking"
+
+const stakeTypeToDaapHref: Record<StakeType.KEEP | StakeType.NU, ExternalHref> =
```

Comment:
Non-blocking:

Typo

```suggestion
const stakeTypeToDappHref: Record<StakeType.KEEP | StakeType.NU, ExternalHref> =
```

Reply:
b3b450c